helm_deploy/django-app/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "django-app.fullname" . }}
+type: Opaque
+data:
+  secretKey: "ZEdocGMybHpWR2hsVTJWamNtVjBTMlY1VG05MGFHbHVaMVJ2VTJWbFNHVnlaUzQ9"
```

Comment:
The values in this `Secret` object should be set in `values.yml` and interpolated here, so that they can be set at chart install time without having to modify the chart.

Reply:
Would this be a case of having a blank entry for secretKey in values.yaml?

Reply:
I think so - I'm pretty sure Django will complain if that value isn't set, so it'd force users to define their own secret key value. But also, as discussed in person, it'd be good to declare this as a [required value](https://github.com/kubernetes/helm/blob/master/docs/charts_tips_and_tricks.md#know-your-template-functions) so that Helm makes sure a value has been set.